apiserver/pkg/util/config.go

```diff
@@ -30,4 +30,7 @@ const (
 
 	// The component name for apiserver
 	ComponentName = "kuberay-apiserver"
+
+	// Timeout for apiserver gRPC server
+	GRPCServerDefaultTimeoutSeconds = 60
```

Comment:
curious, why don't we directly use the `time.Duration`? so we don't need to suffix the unit.

Reply:
I prefer timeout duration constants over literal value, because it naturally contains unit